docs/guides/dev_environment.md

```diff
@@ -40,8 +40,12 @@ make prep
 
 # Filling out the config
 Locate `config.default.jsonc`, and create a copy of it named `config.jsonc` in the same folder. 
+Locate `secrets.default.jsonc` and create a copy of it named `secrets.jsonc` in the same folder. 
 
-Open the newly created config file, and copy the newly created auth token into the "authToken" key at the top of the json. Read through the rest of the config file, and edit as you see fit.
+Open `secrets.jsonc` and fill the"discordAuthToken" key near the top of the page with your discord *bot* token 
+
+If you're receiving an error where the bot spams `Value "" is not snowflake.` The most probable cause is your discord ID not being set in `confic.jsonc`. To fix it, fill your discord ID in to use for DM logging. Otherwise set "verboseLevel" to `0` 

```

Comment:
```suggestion
If you're receiving an error where the bot spams `Value "" is not snowflake`, the most probable cause is your discord ID not being set in `config.jsonc`. To fix it, fill your discord ID in to use for DM logging. Otherwise set "verboseLevel" of "directMessageLogging" to `0` .

```